synr/compiler.py

```diff
@@ -252,16 +252,29 @@ def compile_stmt(self, stmt: py_ast.stmt) -> Stmt:
                         {},
                     ),
                 )
-            elif not isinstance(lhs, Var):
+            elif isinstance(lhs, Var):
+                lhs_vars = [lhs]
+            elif isinstance(lhs, ArrayLiteral) or isinstance(lhs, Tuple):
```

Comment:
I think it would be best if we just supported `x,y = ...` instead of `[x,y] = ...` (note that supporting the former implies that we support `(x,y) = ...`). Thoughts?